### ISSUE TITLE
Is there good example for onMouseOver\onMouseOut binding?

### ISSUE THREAD

**Issue by americaru** [littlebits/react-popover#76]

**Comment by ilyasf** [comment]
I have the same question and only could resolve for myself by the following way
1 - Write wrapper for popover body {div} 
2 - Handle onMouseOver/Enter in the wrapped div
3 - add css styles {overflow-y: auto; width: 100%; height: 100%; padding-right: 20px} for the wrapper

**Comment by jasonkuhrt** [comment]
I'm not sure I fully understand what you are asking for here. Are you wondering how to use `onMouseOver` and `onMouseOut` on the popover body?

**Comment by rafatwork** [comment]
@jasonkuhrt 
Question is about closing the popover when the mouse leaves the popover body and does not hover the popover trigger.

**Comment by jasonkuhrt** [comment]
It seems like you should be able to just pass `onMouseOver` et al to https://github.com/littlebits/react-popover#standard and voila. Is that not working? Am I misunderstanding the ask?

**Comment by rafatwork** [comment]
@jasonkuhrt 
I can't get events such as onMouseOver  et al to work on the <Popover />-element. Can you?

If you hover over the triggering element, the popover body appears. If you then move your mouse to the popover body, it is still displayed. Only when you move your mouse outside the popover body, it disappears.

With onOuterAction, you can make the popover body disappear easily when clicked outside the popover body. But it isn't triggered when the mouse just moves outside the popover body.

**Comment by rafatwork** [comment]
[I eventually managed to get a working solution myself.](https://gist.github.com/rafatwork/2ccc008bd3bcd1a721e083ce001eebfa)
This does the job for me, but improvements/comments are always welcome,

**Comment by jasonkuhrt** [comment]
> But it isn't triggered when the mouse just moves outside the popover body.

Right, because `onOuterAction` is defined as a click/tap, not simply movement.

Are you saying you want a prop like `onOuterMovement`?

**Comment by rafatwork** [comment]
@jasonkuhrt 
Exactly!

**Comment by MrBlenny** [comment]
Have a look at [react-popover-wrapper](https://github.com/MrBlenny/react-popover-wrapper). This is a wrapper component I use to easily allow click, hover and hoverDelay triggers.

**Comment by jasonkuhrt** [comment]
@MrBlenny Cool. I'm glad I clarified the issue. As for ETA I don't know. Right now my spare-time efforts are largely going into https://github.com/jasonkuhrt/forto.

**Comment by kartikluke** [comment]
Took a stab at a solution here #163.
